{
    "priority": "ERROR",
    "payload": {
        "$ref": "common_payloads/InstanceActionRebuildPayload.json#",
        "nova_object.data": {
            "fault": {
                "nova_object.name": "ExceptionPayload",
                "nova_object.data": {
                    "module_name": "nova.tests.functional.notification_sample_tests.test_instance",
                    "exception_message": "Virtual Interface creation failed",
                    "function_name": "_virtual_interface_create_failed",
                    "exception": "VirtualInterfaceCreateException",
                    "traceback": "Traceback (most recent call last):\n  File \"nova/compute/manager.py\", line ..."
                },
                "nova_object.version": "1.1",
                "nova_object.namespace": "nova"
            },
            "architecture": null,
            "image_uuid": "a2459075-d96c-40d5-893e-577ff92e721c",
            "task_state": "rebuilding"
        }
    },
    "publisher_id": "nova-compute:compute",
    "event_type": "instance.rebuild.error"
}
